redirects/wiki-crm.txt

```diff
@@ -1,5 +1,5 @@
 Documentation+Infrastructure+Canary develop
 Book+style+guide best-practices/documentation-style-guide
-CiviCRM+Unit+Testing+basic+information testing/setup
```

Comment:
I updated this (although it's not quite clear in the way Github highlights this line).

Reply:
I'm taking this as a lesson in Github mechanics -- when there's a line edit (removal+addition), you can comment on either the removal or the addition. It seems better to comment on the addition -- because (in that case) Github will automatically recognize that the code-review has been addressed.

Reply:
Ahh nice point! Noted.

Also worth mentioning that I really like the way Git*Lab* allows you to "resolve" separate discussions manually within merge requests. Very slick.